infrastructure/main.tf

```diff
@@ -6,13 +6,13 @@ module "submit-your-appeal-frontend" {
   ilbIp    = "${var.ilbIp}"
 
   app_settings = {
-    TRIBUNALS_CASE_API_URL       = "http://sscs-tribunals-api-${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+    TRIBUNALS_CASE_API_URL       = "${var.tribunals_case_api}"
```

Comment:
This URL will be populated based on environment. So no need to change it and duplicate it multiple files.

Reply:
it differs on sandbox and production subscription. tfvars file refers to previous template

Reply:
it has to be unfortunately. it is a bit different for sandbox at the moment - we are not using saat yet on sandbox - that's why there is a difference in api url later